src/app/components/shell/header/header.js

```diff
@@ -119,25 +119,62 @@ class Header extends BaseView {
 
     toggleFullScreenNav(e) {
         let button = e.currentTarget;
-        let menus = this.el.querySelector('.menus');
+        let header = this.el.querySelector('.page-header');
 
-        menus.classList.toggle('hidden');
         document.body.classList.toggle('no-scroll');
 
         window.requestAnimationFrame(() => {
+            header.classList.toggle('active');
+            this.removeAllOpenClasses(e);
+
             button.classList.toggle('expanded');
             button.setAttribute('aria-expanded', !!button.classList.contains('expanded'));
         });
     }
 
+    removeClass(array, className) {
+        let i = 0,
+            len = array.length;
+
+        for (; i < len; i++) {
```

Comment:
Note: you can use the `for (let item of array)` style for loop for cleaner loops.
Not a blocker, just information.